CONTENT.md

```diff
@@ -0,0 +1,280 @@
+
+To add a hosting provider entry, add the following template to the `providers.toml`. Logos should be placed next to the the toml file. A logo should be in the SVG format. However if this is not available a PNG is acceptable provided it has sufficient resolution while also having reasonable filesize.
+
+```toml
+[[providers]]
+name = "The provider's name"
+image = "logo.svg"
+website = "https://example.com"
+operating_since = "2018"
+description = """
+What the service provides and any noteworthy limitations or features.
+It should NOT contain blatant marketing text. We hold the right to ask for rewording it or reject it.
+"""
+``` 
+
+## Adding documentation
+
+Matrix.org provides a place to have documentation. Documentation is required to be on topic of the Matrix Protocol itself or how to get started using the protocol.
+
+We currently split the Documentation into these types:
+
+- Onboarding Documentation - Documentation for first time users
+- Community Documentation - Documentation which we consider is helpful to run a good community using Matrix tools. This can include guidance on things like moderation or how Matrix features benefit Communities.
+- Technical Documentation - Documentation that help people understanding to Matrix Protocol. This can range from a rough outline on how Matrix functions to guides on things like state resolution.
+
+Additionally to creating a documentation there comes a responsibility to the author. We require authors of a documentation page to be reachable or delegate a new contact point if they cant maintain it any further. We will tr at least once per year to reach out to an author to verify a documentation is still accurate. This may happen more often when we notice ourself that the documentation is likely stale. If there hasn't been a reply in a reasonable timeframe we hold the right to archive a documentation page without further warning as we cannot verify all documentation on a technical level. Therefor we rather list up to date maintained documentation rather than possibly stale documentation. These archived documentations can then be found at the [Older Documentation Section](https://matrix.org/docs/older/#all-past-documentation).
+
+### What we expect from Documentation contributions
+
+Documentation on matrix.org should be documentation which is part of the 3 categories we defined above. If you think that there is a category missing then please first open an issue for us to discuss this.
+
+The Documentation is expected to have a writing style. This means specifically that we expect the documentation to be written in a neutral and personal tone. This means we prefer "To do X you want to do Y" over "To get to X one should do Y". Additionally to the tone we also expect a documentation page to be structured. think about headers to use which guide the reader and make it easy to find a section in the document.
```

Comment:
```suggestion
```

I would refrain from adding a paragraph about it until we have proper guidelines on the voice and tone :)